### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by johnie261**
I did the following activities:

- Follow Github Flow.
- Implemented Webpack and Webpack dev server.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #d3d3d3;
+  align-items: center;
+  padding: 0 1rem;
+}
+
+.list-title h3 {
+  font-size: 1.5rem;
+}
```
Comment (Shedrack-Sunday) [inline src/style.css:25]: - [ ] Nice work you did on the project 👏 . However, there seems to be a little disparity between your design and that given the project requirements. Kindly find the difference in the structure below, and effect the changes appropriately 👍 

## Your design :
image


## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @johnie261    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
  
Required Changes ♻️

- [x] Nice efforts so far. The little changes let have to do with the structure of rendering the content dynamically which can be fixed with a bit of styling. 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/style.css (hunk 1 above)
Comment (johnie261) [reply src/style.css:25]: Hey Shedrack, I really don't understand what you mean. Since when I run npm start it displays the suggested templated above

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @johnie261 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**